### PR TITLE
Initial typing in RequestGroup description no longer has error

### PR DESCRIPTION
**Pull request by guy-ro**
Now, when someone first clicks on RequestGroup description in a request group form, no error is shown. Fixed some general errors and mistakes in related code

Removed some console.log someone left

### CODE REVIEW

File: client/src/components/atoms/RichTextField.tsx [hunk 1]
```diff
@@ -49,30 +50,39 @@ const RichTextField: FunctionComponent<Props> = (props: Props) => {
     }
 
     function onChange(state: EditorState) {
-        if (!changeMade) {
-            setChangeMade(true)
-        }
+        // current component state
+        let isEmpty = empty
+        let hasChangeMade = changeMade
+        console.log("Empty: " + isEmpty + " | ChangeMade: " + hasChangeMade)
```
Comment (chamod-gamage) [inline client/src/components/atoms/RichTextField.tsx]: stray log

**Review by chamod-gamage — APPROVED** [review]
Awesome work - LGTM! Just a stray log to take care of

File: client/src/components/atoms/RichTextField.tsx (hunk 1 above)
Comment (guy-ro) [reply client/src/components/atoms/RichTextField.tsx]: done!

File: client/src/components/atoms/RichTextField.tsx [hunk 2]
```diff
@@ -155,7 +164,7 @@ const RichTextField: FunctionComponent<Props> = (props: Props) => {
             </div>
             <div className="richtext-field-input">
                 {/* Do not display default text if there is initial content and no change is made */}
-                {!active && !(props.initialContent && !changeMade) &&
+                {changeMade && !active && !props.initialContent && empty &&
```
Comment (meganniu) [inline client/src/components/atoms/RichTextField.tsx:167]: Would it be possible to keep `!(props.initialContent && !changeMade)`? It matches the comment on line 166

**Review by meganniu — APPROVED** [review]
LGTM!

**Pull request merged**